### PR TITLE
FT: Functional tests kms 

### PR DESCRIPTION
**Pull request by alexandre-merle**
- Create encryption bucket in s3cmd functionals tests if
  the env variable ENABLE_KMS_ENCRYPTION is set to true
- Update circle.yml to test all cases

### CODE REVIEW

File: tests/functional/s3cmd/tests.js [hunk 1]
```diff
@@ -169,12 +230,25 @@ describe('s3cmd putBucket', () => {
...
     });
 
     it('should delete bucket put with region', done => {
-        exec(['rb', `s3://regioned`, '--region=wherever'], done);
-    });
+        exec(['rb', 's3://regioned', '--region=wherever'], done);
+    });
+
+    if (process.env.ENABLE_KMS_ENCRYPTION === 'true') {
+        it('create a valid bucket with server side encryption',
```
Comment (LaurenSpiegel) [inline tests/functional/s3cmd/tests.js]: creates

**Comment by LaurenSpiegel** [comment]
👍

**Comment by alexandre-merle** [comment]
@ironman-machine let's do that

File: circle.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 general:
   branches:
     ignore:
-      - /^ultron\/.*/ # Ignore ultron/* branches
+      - /^ultron\/.*/   # Ignore ultron/* branches
```
Comment (RachedBenMustapha) [inline circle.yml]: Was this space intended?

Comment (alexandre-merle) [inline circle.yml]: yes, to remove a warning in the yml file ^^

**Comment by alexandre-merle** [comment]
End to end passing :)

**Comment by nicolas2bert** [comment]
👍

**Pull request merged**